README.md

```diff
@@ -19,19 +19,61 @@ it ideal for a demo/template.
 - Riverpod Provider Generation
 - Retrofit API calls, `toJson` function Generation.
 
-## How to run
+For help getting started with Flutter development, view the
+[online documentation](https://docs.flutter.dev/), which offers tutorials, samples, guidance on
+mobile development, and a full API reference.
 
+## How to build
 - run below command
-
 ```
 flutter pub get
 flutter packages pub run build_runner build --delete-conflicting-outputs
 ```
 
-### Visual Studio
+## Run 
+### Android Studio
+Launch Configurations are included in the `.run` folder. 
 
-Launch Configurations are included in the `.vscode/launch.json` file.
 
-### Android Studio
+### Run iOS/Android using Visual Studio
+- create launch.json and add below
```

Comment:
Since we will commit the `launch.json` to the repository, this part of the readme (create json) is not necessary.

Reply:
nice idea fixed.